src/web/app/pages-admin/admin-notifications-page/admin-notifications-page.component.html

```diff
@@ -3,14 +3,17 @@ <h1>Notifications</h1>
 
 <div>
   <button id="btn-add-notification" class="btn btn-success" (click)="isNotificationEditFormExpanded = true"
-          [disabled]="isNotificationEditFormExpanded || isNotificationLoading"><i class="fas fa-plus"></i> Add New Notification
+          [disabled]="isNotificationEditFormExpanded || isNotificationLoading">
+          <span *ngIf="currentNotificationEditFormMode === NotificationEditFormMode.ADD else editbtn"><i class="fas fa-plus"></i> Add New Notification</span>
+          <ng-template #editbtn><i class="fas fa-cog"></i> Edit Existing Notification</ng-template>
   </button>
 </div>
 
 <div class="margin-top-30px" *ngIf="isNotificationEditFormExpanded && !isNotificationLoading" @collapseAnim>
-  <tm-notification-edit-form [formMode]="NotificationEditFormMode.ADD" [(model)]="notificationEditFormModel"
+  <tm-notification-edit-form [formMode]="currentNotificationEditFormMode" [(model)]="notificationEditFormModel"
                         (addNewNotificationEvent)="addNewNotificationHandler()"
-                        (closeEditFormEvent)="isNotificationEditFormExpanded = false"
+                        (editExistingNotificationEvent)="editExistingNotificationHandler()"
+                        (cancelEditingNotificationEvent)="initNotificationEditFormModel()"
   ></tm-notification-edit-form>
```

Comment:
Just noticed this, could you put the right angled bracket on the previous line to close the component